### PR TITLE
pd-ctl: Update pd-ctl document

### PR DESCRIPTION
**Pull request by lhy1024**
- [x] config show and delete
- [x] store limit and scene
- [x] placement-rules config 
- [x] remove label
- [x] member output  
- [x] scheduler resume and pause
- [x] hot scheduler config

### CODE REVIEW

File: tools/pd-ctl/README.md [hunk 1]
```diff
@@ -123,52 +125,27 @@ Usage:
...
-  "location-labels": ""
+  "location-labels": "",
+  "strictly-match-label": "false",
+  "enable-placement-rules": "false"
 }
 >> config show cluster-version                // Display the current version of the cluster, which is the current minimum version of TiKV nodes in the cluster and does not correspond to the binary version.
-"2.0.0"
+"4.1.0-alpha"
+
+>> config delete namespace <name> [flags]
```
Comment (rleungx) [inline tools/pd-ctl/README.md:146]: We have already remove namespace

File: tools/pd-ctl/README.md [hunk 2]
```diff
@@ -552,37 +558,104 @@ Usage:
...
+>> store 1                      // Get the store with the store id of 1
   ......
->> store delete 1                      // Delete the store with the store id of 1
+>> store delete 1               // Delete the store with the store id of 1
   ......
->> store label 1 zone cn               // Set the value of the label with the "zone" key to "cn" for the store with the store id of 1
->> store weight 1 5 10                 // Set the leader weight to 5 and region weight to 10 for the store with the store id of 1
->> store remove-tombstone              // Remove stores that are in tombstone state
->> store limit                         // Show limits of adding region operation for all stores
->> store limit region-add              // Show limits of adding region operation for all stores
```
Comment (rleungx) [inline tools/pd-ctl/README.md:577]: Why remove these?

File: tools/pd-ctl/README.md [hunk 3]
```diff
@@ -597,6 +670,23 @@ system:  2017-10-09 05:50:59 +0800 CST
 logic:  120102
 ```
 
+### dynamic config
```
Comment (rleungx) [inline tools/pd-ctl/README.md:673]: This part will be removed in 4.0

File: tools/pd-ctl/README.md [hunk 4]
```diff
@@ -284,6 +261,34 @@ This option only works when key type is "table".
...
+
+>> config placement-rules show --group=pd --id=default // Display placement-rule in pd group and with id.
+
+>> config placement-rules show --region=2 // Display placement-rule with region
+
+>> config placement-rules save --in=rules.json // Set rules with rules.json
+
+>> config placement-rules load --group=pd --out=rule.txt // Output rules to `rule.txt`
+```
+
```
Comment (rleungx) [inline tools/pd-ctl/README.md:289]: remove the extra lines

File: tools/pd-ctl/README.md [hunk 5]
```diff
@@ -552,37 +558,104 @@ Usage:
...
+    ```bash
+    >> scheduler config balance-hot-region-scheduler set byte-rate-rank-step-ratio 0.05
+    ```
+
+- `src-tolerance-ratio` and `dst-tolerance-ratio` are config for expectation scheduler. The smaller the `tolerance-ratio` , the easier it is to schedule. When redundant scheduling occurs, we can appropriately increase this.
+
+    ```bash
+    >> scheduler config balance-hot-region-scheduler set src-tolerance-ratio 1.05
+    ```
+
```
Comment (rleungx) [inline tools/pd-ctl/README.md:625]: ditto

File: tools/pd-ctl/README.md (hunk 1 above)
Comment (lhy1024) [reply tools/pd-ctl/README.md:146]: ok, and we need to delete it from help echo

File: tools/pd-ctl/README.md [hunk 6]
```diff
@@ -284,6 +259,32 @@ This option only works when key type is "table".
 
 - `enable-location-replacement` is used to enable the isolation level check. When you set it to `false`, PD does not improve the isolation level of Region replicas by scheduling.
 
+- `enable-debug-metrics` is used to enable the debug metrics. When you set it to `true`, PD will open some metrics, such as balance-tolerant-size and op influence.
+
+- `store-limit-mode` has two mode for setting limit: auto or manual, an auto-set value can be overwritten by a manual-set value, otherwise it is forbidden.
+
+#### Placement-rules
+
+[Placement Rules](<https://pingcap.com/docs-cn/stable/how-to/configure/placement-rules/#placement-rules-%E4%BD%BF%E7%94%A8%E6%96%87%E6%A1%A3>) is region rules system used to guide PD to generate corresponding schedules for different types of data.
```
Comment (disksing) [inline tools/pd-ctl/README.md:268]: better to refer english version

File: tools/pd-ctl/README.md [hunk 7]
```diff
@@ -406,14 +408,14 @@ Usage:
...
-### `region key [--format=raw|pb|proto|protobuf] <key>`
+#### `region key [--format=raw|pb|proto|protobuf] <key>`
 
 Use this command to query the region that a specific key resides in. It supports the raw and protobuf formats.
 
 Raw format usage (default):
 
 ```bash
->> region key abc
+>> region key abcd // avoid to use odd length hex string
```
Comment (disksing) [inline tools/pd-ctl/README.md:418]: user may wonder why can't use old length... maybe just `// use hex format`

**Comment by lhy1024** [comment]
@nolouch @rleungx @disksing PTAL

**Pull request merged**